staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -154,19 +191,26 @@ func (m *Helper) Patch(namespace, name string, pt types.PatchType, data []byte,
 
 func (m *Helper) Replace(namespace, name string, overwrite bool, obj runtime.Object) (runtime.Object, error) {
 	c := m.RESTClient
+	var options = &metav1.DeleteOptions{}
```

Comment:
Why is this `DeleteOptions`? I think it should be `UpdateOptions`.

Reply:
Oops, done.